openmdao/core/problem.py

```diff
@@ -1470,6 +1471,21 @@ def check_totals(self, of=None, wrt=None, out_stream=_DEFAULT_OUT_STREAM, compac
 
         # TODO: Once we're tracking iteration counts, run the model if it has not been run before.
 
+        if wrt is None:
+            wrt = list(self.driver._designvars)
+            if wrt is None:
```

Comment:
Did you mean to put `if not wrt:` here instead? 'wrt' can never be None here.  Same goes for `if of is None:` check below.

Reply:
oops, I missed these here. Fixed